lessonplans/1_BayesianRefresher.md

```diff
@@ -0,0 +1,54 @@
+# Section 1: Bayesian refresher and Introduction to ArviZ
```

Comment:
I'd suggest also having a 10-15 min introduction:

- "bayesianism vs frequentism" as "uncertain parameters vs uncertain data"
- MCMC as a way of computing expectations
- In the context of MCMC, a "distribution" is a collection of samples, possibly with more structure

Reply:
Good points, will add all these talking points